src/components/select/select.scss

```diff
@@ -120,7 +142,12 @@ md-select {
     // to create a dotted line under the input.
     background-size: 4px 1px;
     background-repeat: repeat-x;
-    margin-bottom: -1px; // Shift downward so dotted line is positioned the same as other bottom borders
+    // Add to padding-bottom to keep dotted line aligned with other bottom borders
+    // Sub from padding-top to keep height consistent
+    // Translate text 1px up to keep in alignment
+    padding-bottom: $select-value-padding-bottom + 1;
+    padding-top: $select-value-padding-top - 1;
+    transform: translate3d(0, 1px, 0);
```

Comment:
`translate3d` generally breaks IE. Can just `translate` be used here instead?

Reply:
I'm just moving this from what was Line#160, so it's unlikely to cause any major rendering changes.

You're probably right that 2D translation is probably better, but I think that would be outside the scope of the PR. It's best not to bunch too many changes at once.